cmstestsuite/unit_tests/db/filecacher_test.py

```diff
@@ -361,18 +364,15 @@ class TestFileCacherDB(TestFileCacherBase, DatabaseMixin, unittest.TestCase):
     """Tests for the FileCacher service with a database backend."""
 
     def setUp(self):
-        super().setUp()
-        file_cacher = FileCacher()
-        self._setUp(file_cacher)
+        super(DatabaseMixin).setUp()
+        super(TestFileCacherBase).setUp(FileCacher())
 
 
 class TestFileCacherFS(TestFileCacherBase, unittest.TestCase):
     """Tests for the FileCacher service with a filesystem backend."""
 
     def setUp(self):
-        super().setUp()
-        file_cacher = FileCacher(path="fs-storage")
-        self._setUp(file_cacher)
+        super().setUp(FileCacher(path="fs-storage"))
```

Comment:
It's a bit suspicious codecov says this line (and other similar ones) are not covered by tests.

Reply:
Indeed. I'm looking into it.

Reply:
I think this was caused by `__test__ = False` on the parent class (TestFileCacherBase) being inherited by TestFileCacherDB and TestFileCacherFS. Fixing this made me realize that `super(ParentClass).setUp()` doesn't work like I though it did 😅  so now I'm using `ParentClass.setUp(self)` which although ugly seems to work.